src/Config/ObjectModel/OpenTelemetryOptions.cs

```diff
@@ -0,0 +1,7 @@
+namespace Azure.DataApiBuilder.Config.ObjectModel;
+
+/// <summary>
+/// Represents the options for configuring Open Telemetry.
+/// </summary>
+public record OpenTelemetryOptions(bool Enabled = false, string? Endpoint = null, string? Headers = null, string? ServiceName = null)
```

Comment:
Should `Endpoint` be nullable?

Reply:
I referenced the same logic we have in the ApplicationInsightsOptions, in which the ConnectionString is nullable